runtime/development/src/lib.rs

```diff
@@ -1914,6 +1917,24 @@ impl_runtime_apis! {
 		}
 	}
 
+	// RewardsApi
+	impl runtime_common::apis::RewardsApi<Block, AccountId, Balance, RewardDomain, CurrencyId> for Runtime {
+		fn list_currencies(account_id: AccountId) -> Option<Vec<(RewardDomain, CurrencyId)>> {
+			let currencies = pallet_rewards::Pallet::<Runtime, pallet_rewards::Instance1>::list_currencies(account_id);
+
+			Some(currencies)
```

Comment:
It seems like we have no error handling happening here so we always return `Some(x)`. Why not just make the return type `Vec(RewardDomain, CurrencyId)`?

Reply:
I did that to keep it consistent with the rest of the APIs where we always return options.

Reply:
Yeah ok that's fine 👍

Reply:
OK, I went ahead and dropped the option. Thank you both for the input on this!